### PR TITLE
USWDS Compile 1.2.0

### PR DESCRIPTION
**Pull request by thisisdano**
# What's new in USWDS Compile 1.2.0

## Features
- **Updated Gulp to version 5.** This removes the dependency vulnerabilities related to Gulp 4. (#96)
- **Added a setting to turn off Sass sourcemaps.** Now, you can set the 
new `settings.compile.sassSourcemaps` to `false` in your `gulpfile.js` to prevent the system from generating CSS sourcemaps. Thanks @mdmower-csnw! (#103)

## Dependencies and security

| Dependency name | Previous version | New version |
| :-------------- | :--------------- | :---------- |
| autoprefixer    | 10.4.16          | 10.4.20     |
| gulp            | 4.0.2            | 5.0.0       |
| gulp-sourcemaps | 3.0.0            | --          |
| postcss         | 8.4.31          | 8.4.40      |
| sass-embedded   | 1.69.5           | 1.77.8      |
| @uswds/uswds    | 3.7.0            | 3.8.1       |

`0` vulnerabilities using `npm audit`.

Release TGZ SHA-256 hash: `53995f42af77dcf5a7f66e5adebeebf2742a662d3cf7a12587b38b22422db122`

### CODE REVIEW

File: .snyk
Comment (amyleadem) [inline .snyk:1]: @mahoneycm @mejiaj Is this file still needed? It looks like all the ignores have expired but the tests are still passing.

**Review by amyleadem — COMMENTED** [review]
Looking good to me! Just one open comment about the necessity of the `.snyk` file. 

- [x] Confirmed I could successfully install, start, and run gulp commands with this branch in uswds-site with uswds v3 and uswds-sandbox with v2
- [x] Confirmed the content in the dependency tables in the PR description and release notes match the dependency changes in package.json
- [x] Confirmed the items tagged for 1.2.0 have been merged into `develop` and that this branch is up to date with `develop`
- [x] Confirmed all references to "Gulp 4" have been updated

**Comment by thisisdano** [comment]
Made a change on this branch, so updated SHA 256 hash: `53995f42af77dcf5a7f66e5adebeebf2742a662d3cf7a12587b38b22422db122`

File: .snyk
Comment (mahoneycm) [reply .snyk:1]: File removed in [9a9d4cd](https://github.com/uswds/uswds-compile/pull/117/commits/9a9d4cdf0650e626ccf3ce078d3e3474326febd3)

**Review by amyleadem — APPROVED** [review]
Looks good to me! Confirmed `.snyk` was successfully removed and the SHA 256 hash in the .tgz matches the release notes.

**Comment by mahoneycm** [comment]
RE: The `snyk` page, I know we don't have a script to run snyk tests on this repo _but_ running `npx snyk test` does result in the following error:

image

Flagging in case we want to include snyk tests here
